scripts/g.bands/g.bands.html

```diff
@@ -0,0 +1,175 @@
+<h2>DESCRIPTION</h2>
+
+Prints band references information related to image collections.
+
+<p>
+Band reference (a shortcut or fully defined band reference identifier)
+is specified by <b>band</b> option. Extended metadata is printed only
+when <b>-e</b> flag is given.
+
+<h3>Image collections</h3>
+
+Image collections are the common data type to reference time series of
+multi band data. It is used in many frameworks
+(see <a href="https://developers.google.com/earth-engine/tutorial_api_04">Google
+Earth Engine API</a> for example) to address multi spectral
+satellite images series. GRASS supports a multi-band raster layer
+approach rudimentarily with the imagery group concept
```

Comment:
I don't think the word "rudimentarily" should be here. Either it is *the* way of doing image collections in GRASS or it is not.

Reply:
Changed in 60d1289e1b